test/lifecycle_test.dart

```diff
@@ -362,6 +375,7 @@ void sharedLifecycleTests<T extends react.Component>({
         expect(
             component.lifecycleCalls,
             equals([
+              matchCall('getChildContext'),
```

Comment:
Is this left over from a bad merge with Keal's context PR?

Reply:
Same note as above!